src/common/snippets/include/snippets/op/horizon_sum.hpp

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include "openvino/op/op.hpp"
+#include "snippets/op/horizon_max.hpp"
```

Comment:
Now it should be `snippets/lowered/shape_inference/shape_inference.hpp` and in source file.

Reply:
Removed the header, thanks. Again, no header needed in horizon_max.cpp, it's all in shape_infer_instances